src/assets/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "container": "container",
+  "container": "grid-container",
```

Comment:
I would just remove this line completely. We should move to inline classes to better handle possible one off things.